cidc_api/models.py

```diff
@@ -636,6 +637,7 @@ def create_from_metadata(
         assay_type: str,
         file_metadata: dict,
         session: Session,
+        additional_metadata: dict = {},
```

Comment:
You don't want to do mutable values as defaults in python =) 
https://lmgtfy.com/?q=python+argument+default+dict